public/slick/config.rb

```diff
@@ -0,0 +1,10 @@
+css_dir = "."
+sass_dir = "."
+images_dir = "."
+fonts_dir = "fonts"
+relative_assets = true
+
+output_style = :compact
+line_comments = false
+
+preferred_syntax = :scss
```

Comment:
Is there really any need for this file? Or any of the other Slick resources? Don't commit what you aren't going to use.